src/functions.py

```diff
@@ -1,12 +1,23 @@
 # Write a function is_even that will return true if the passed-in number is even.
 
 # YOUR CODE HERE
+def is_even(x):
+    if x % 2 == 0:
+        return True
+    else:
+        return False
 
+print(is_even(2))
+print(is_even(3))
 # Read a number from the keyboard
 num = input("Enter a number: ")
 num = int(num)
 
 # Print out "Even!" if the number is even. Otherwise print "Odd"
 
 # YOUR CODE HERE
+if is_even(num) == True:
```

Comment:
remember truthy and falsy values from javascript? they work in Python too.
you can simply do 
   if is_even(num):
         #do something